### DISCUSSION TITLE
Cannot log in, meet 404 at the 2fa page

### DISCUSSION THREAD

**Discussion by e-yi** [community/community#23442] [New to GitHub] [answered]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/e/7/e77a7a37ad1264233da9a8f333142349fa78b74d.jpeg" data-download-href="/uploads/short-url/x1KHxIfaavL6mBLuDm8MJO1SZVH.jpeg?dl=1" title="404" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/20715359/181077107-1dcc81f8-14f4-400d-a0bc-06c204838121.jpeg" alt="404" data-base62-sha1="x1KHxIfaavL6mBLuDm8MJO1SZVH" width="690" height="356" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/7/e77a7a37ad1264233da9a8f333142349fa78b74d_2_690x356.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/e/7/e77a7a37ad1264233da9a8f333142349fa78b74d_2_1035x534.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/e/7/e77a7a37ad1264233da9a8f333142349fa78b74d_2_1380x712.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/7/e77a7a37ad1264233da9a8f333142349fa78b74d_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">404</span><span class="informations">2397×1239 149 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>what happend:</p>
<ol>
<li>click sign in</li>
<li>enter the correct username &amp; password</li>
<li>the browser jumps to <a href="https://github.com/sessions/verified-device" rel="noopener nofollow ugc">Page not found · GitHub · GitHub</a>
</li>
<li>get the verication code, but no place to enter</li>
</ol>

**Comment by jfullstackdev** [comment]
<p>try logging in in Incognito, if that is not working then</p>
<p>try contacting support</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://support.github.com/contact/account">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/47092464/181077108-2a18a842-29c2-4459-b874-570407719264.png" class="site-icon" width="32" height="32">

      <a href="https://support.github.com/contact/account" target="_blank" rel="noopener">support.github.com</a>
  </header>

  <article class="onebox-body">
    

<h3><a href="https://support.github.com/contact/account" target="_blank" rel="noopener">Accounts - GitHub Support</a></h3>

  <p>GitHub Support is here to help. Learn about GitHub products, browse our helpful resources, and contact support with your questions.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by e-yi** [comment]
<p>fixed by using Chrome’s private browsing mode, and disable all my browser extensions.</p>

**Comment by tuves** [comment]
<p>Thanks for coming back to update with your solution, <a class="mention" href="/e-yi">@e-yi</a>!</p>

**Comment by xCONFLiCTiONx** [comment, marked as answer]
Delete the github cookie and retry. I still hit a 404 but it was after putting in the 2fa code and still logged me in. Logging in using private browser is only good for a one time session.

**Reply by alihassanayedh-pixel** [reply]
طيب حسنا

**Reply by EiJackGH** [reply]
Delete this cookie and retry the 404 page, Turn on Incognito or Private Browsing.

**Comment by tombrau** [comment]
I am having the same issue but with firefox. Setup 2fa as either the app or as an sms it shows the image as seen above. I can login if 2fa is disabled or via a private window. Anyone with any thoughts
cheers
tom

**Comment by ntraft** [comment]
Logging in via Incognito mode should not be an "answer" to this question.

Here is my answer: Clear cookies AND restart Chrome.
- My Chrome was awaiting an update, so I restarted Chrome to allow it to update. After this, login worked.
- I also cleared github.com cookies, but this alone wasn't enough, and I'm not sure it was necessary at all. The restart was the key step.

**Comment by ghost** [comment]
This is Test AnsWer

**Comment by ISTPdev** [comment]
For me it wouldn't let me in unless I verified my device, but the Verify Device page went to 404. I was looking around, I had to spin up Chrome, on a second machine before I realized it's only in Firefox that this page didn't show. I logged in through Incognito. Which is absurd. I only found this on Google using a bunch a keywords, now I'm here to: **WTF amirite?** :confused:

**Comment by Ant0nRocket** [comment]
Solution for Yandex Browser (Chrome based browser from Yandex).
You don't need to use Incognito mode, just install some cookie management add on (like Cookie-Editor or something like that), go to github.com and delete all cookies for that domain. Refresh page. Try sign in again. It worked for me, 2FA code request shown.
I think that the problem github cookies, stored in browser before 2FA activation.  

----

Same in russian: решение для Яндекс.Браузера.
Не нужен вам режим инкогнито, просто порежьте куки github.com каким-нибудь менеджером куки (типа Cookie-Editor или что-то в этом духе) или грохните все куки (но тогда вылетит авторизация на всех других сайтах) и попробуйте войти снова. У меня сработало, после ввода логина и пароля появилось окно ввода кода 2FA.
Очевидно, проблема была в старых куки, сохраненных в браузере ещё до активации двухфакторной авторизации.

**Comment by Ruslanjan1987** [comment]
Hi how are you???help me please 404 page not found github my gmail amangeldijan0503@gmail.com  please

**Comment by defxsec** [comment]
Tengo el mismo problema, puedo acceder a mi cuenta pero no es visible mis proyectos para otras personas, y creo que desde que añadi el 2FA sucedio eso alguno puede ayudarme 
![imagen](https://github.com/community/community/assets/109879632/0354ca4f-d08e-4f2e-a560-0df66a6afa85)

**Reply by EiJackGH** [reply]
Delete this cookie and retry the 404 page, Turn on Incognito or Private Browsing, Sign in again or Sign up for an account.

**Comment by Paulo4lvarez** [comment]
This thing is happening to me on Zen Browser as well. I wish there was another solution other than "deleting cookies", because I'm just trying to add my secondary account.

**Reply by ogios** [reply]
zen-browser workspace can deal with it, things are separated
EDIT: no, it's the container that does the job

**Reply by gregorylearns** [reply]
what worked for me was making sure i was logged in to github first before clicking the site's "Login with GitHub" button
